crowdin.yml

```diff
@@ -1,3 +1,3 @@
 files:
   - source: /messages/en.json
-    translation: /messages/%two_letters_code%.json
+    translation: /%original_path%/%two_letters_code%.%file_extension%
```

Comment:
I'm not sure I understand the issue with the original setting using `/messages/%two_letters_code%.json`?

Reply:
We are doing this to all apps, every time we change something. The idea to make the process as standardized and generic as possible, but the way it works won't change, only the yml file.